app/src/main/java/com/deadlineshooters/yudemy/activities/CourseDetailActivity.kt

```diff
@@ -201,33 +189,20 @@ class CourseDetailActivity : AppCompatActivity() {
 
 
 
-        userRepository.isInCourseList(course.id) { isInCourseList ->
```

Comment:
sao xóa mất nguyên chỗ này rồi. Bữa tui add này vì nếu khóa đó của chính người dùng luôn thì thể hiện "Go to course" luôn chứ ko có cho buy hay enroll gì hết. 
Ví dụ khóa này của chính tui tạo, mà thay gì button enroll now thì là "Go to course" 
![image](https://github.com/DeadlineShooters/Yudemy-Mobile/assets/91927045/6a41a7e3-de64-47ba-b8f9-fd14276b9846)